app/assets/stylesheets/partials/_modal.scss

```diff
@@ -0,0 +1,84 @@
+#modalOverlay {
+  background-color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.